packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/upgrade_protocol_version/v0/mod.rs

```diff
@@ -77,6 +77,22 @@ impl<C> Platform<C> {
                 .clear_version_information(Some(transaction), &platform_version.drive)
                 .map_err(Error::Drive)?;
 
+            let platform_version = PlatformVersion::get(current_block_protocol_version)?;
+            let mut cached_fee_version = self.drive.cache.cached_fee_version.write();
+            if let Some((_, &last_fee_version)) = cached_fee_version.iter().last() {
```

Comment:
BTreeMap doesn't preserve insertion order.

Reply:
@shumkov Yes, but the map holds `EpochIndex` as key, so `iter().last()` returns the greatest inserted `EpochIndex`. So we are good.